src/debug/mod.rs

```diff
@@ -0,0 +1,92 @@
+//! `thetawave_lib` debug module
+use bevy::prelude::*;
+use bevy_prototype_debug_lines::*;
+use bevy_rapier2d::prelude::*;
+
+pub fn collider_debug_lines_system(
+    mut lines: ResMut<DebugLines>,
+    player_colliders: Query<(&ColliderPosition, &ColliderShape)>,
+    rapier_config: Res<RapierConfiguration>,
+) {
+    // draw colliders
+    for (collider_position, collider_shape) in player_colliders.iter() {
+        let collider_cuboid = collider_shape.as_cuboid().unwrap();
+        let collider_translation = collider_position.translation;
+
+        draw_debug_collider_cuboid(
+            &mut lines,
+            collider_cuboid,
+            collider_translation,
+            rapier_config.scale,
+        );
+    }
+}
+
+fn draw_debug_collider_cuboid(
```

Comment:
There is probably a good case to replace this with a macro, but I don't know if I'm ready to try that yet.